core/src/toga/window.py

```diff
@@ -337,13 +341,13 @@ def widgets(self) -> Mapping[str, Widget]:
     ######################################################################
 
     @property
-    def size(self) -> tuple[int, int]:
-        """Size of the window, as a tuple of ``(width, height)``, in
+    def size(self) -> Size:
+        """Size of the window, as a ``toga.Size``, in
```

Comment:
There's no need for the "as a" in the getter docstrings, because the return type annotation is already visible in the docs.